generators/server/templates/src/main/java/package/config/SecurityConfiguration_imperative.java.ejs

```diff
@@ -31,7 +31,6 @@ import org.springframework.http.HttpMethod;
 <%_ } _%>
 import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
```

Comment:
I talked with Rob Winch about this annotation, and he mentioned it's only needed when running in a non-Spring Boot app.